### DISCUSSION TITLE
ffmpeg hwaccel for Broadcom Crystal HD

### DISCUSSION THREAD

**Discussion by pekindenis** [blakeblackshear/frigate#13879] [Ask A Question]
Is it possible to use [Broadcom Crystal HD](https://en.wikipedia.org/wiki/Broadcom_Crystal_HD) as Hardware Acceleration? FFmpeg supports this accelerator, but I couldn't find a way to add it in the [documentation](https://docs.frigate.video/configuration/hardware_acceleration/).

**Comment by NickM-27** [comment]
You'd have to check if the ffmpeg binary is built with support for it. Either way you'll need to use manual ffmpeg args

**Reply by pekindenis** [reply]
> You'd have to check if the ffmpeg binary is built with support for it. Either way you'll need to use manual ffmpeg args

How to use manual ffmpeg args ?

**Reply by NickM-27** [reply]
Set hwaccel args with the values required for broadcom
